ci/build.py

```diff
@@ -284,8 +288,10 @@ def script_name() -> str:
                         default=1,
                         type=int)
 
-    parser.add_argument("-c", "--cache", action="store_true",
-                        help="Enable docker registry cache")
+    parser.add_argument("-c", "--no-dockerhub-cache", action="store_true",
```

Comment:
I would rather have it explicit. but we would have to code it if we want it the way you say.

Reply:
The problem is that for "default" behaviour, which I assume is use local cache (as for normal docker builds) we need to provide --no-dockerhub-cache explicitly. I think it should be the other way around.

Reply:
`-c` is not a good shortening anymore for the option

Reply:
The default behavior remains unchanged if I'm not mistaken. Which is to use the dockerhub cache.

Reply:
I'm also confused what the difference between this and just not specifying '--docker-registry' is.

Reply:
You can see in the diff that at least what the option did changed:
```
parser.add_argument("-c", "--cache"
```
```
parser.add_argument("-c", "--no-dockerhub-cache"
```
Previously it was enable and now it is disable. Am I missing something?

Reply:
Previously It was not functioning properly. 
Also check line 308. I think you are missing something as the effect is now negated. The option disables the cache if present. Default is cache enabled. I tested all this.


What single letter abbreviation do you suggest instead?

Reply:
Kellen. One doesn't need to specify the registry. Hence this option to disable cache which is enabled by default.

Reply:
This seems like a work around for broken behaviour to me.  Would you mind if we leave it out of this PR, and try to think about how to fix the root cause of the problem?  To me it makes sense to _not_ use remote caching if a user doesn't specify '--docker-registry'.

Reply:
That’s bad because we want to use cache by default. This was discussed at length with Marco before and we came to agree on this behavior. I wouldn’t want to change that.

Reply:
That’s bad because we want to use cache by default. This was discussed at length with Marco before and we came to agree on this behavior. I wouldn’t want to change that.

Reply:
Ok, that's the information I was missing here.  In this case it makes sense to me.